src/locales/zh.locale.json

```diff
@@ -163,7 +163,9 @@
   "mno_enterprise.templates.dashboard.marketplace.index.title": "市场",
   "mno_enterprise.templates.dashboard.marketplace.index.select_all": "全部",
   "mno_enterprise.templates.dashboard.marketplace.show.back": "返回",
+  "mno_enterprise.templates.dashboard.marketplace.show.conflicting_app": "You cannot install this App as it is incompatible with:",
```

Comment:
I've put the key so that we could translate later. And let in english so that it is not completly weird. I'll remove it.